### DISCUSSION TITLE
Problem with relinking

### DISCUSSION THREAD

**Discussion by JeanSemeur** [latenitefilms/BRAWToolbox#190] [Support] [answered]
Hi,
I'm writing a chapter about your app for my reader (martingosset.com) and there is a problem with relinking synchronised clip to a new disk.
When i use your workflow to relink synchronised clips to another disk, BRAW ToolBox create a new event (called "2-11-2024"). So the option "replace" is not available. While original synchronised clip were in an Event called "RUSHES"...
It's a bit problematic...
So it seems that BrawToolBox doesn't take care about the fact the original clips were in the "RUSHES" event.
Thank for your help.
Martin

**Comment by latenitefilms** [comment]
Have you read this page?

https://brawtoolbox.io/relinking/

Relinking with BRAW Toolbox can be a bit of a pain - I generally recommend just keeping the paths the same, so if you need to restore an old job, just make sure BRAW Toolbox has sandbox access and restore the files to the original path.

**Comment by JeanSemeur** [comment, marked as answer]
Thank Chris,
I think i've followed all the steps indicated and i don't have the "leave in place" or "replace" option because Brawtoolbox creates a NEW EVENT (or FCP doesn't want to consider it as the same "RUSHES" event so it creates a new one), so i think FCP X won't ask me if i want to "replace" or "leave". I think the problem is just with the destination event in the  XML generated by your app.
Note : there is no problem with relinking. The new clips Braw toolbox creates are well relinked to the new disk. No problem with that.

(PS: if i write this chapter in French it's because english is not easy for everyone)

**Reply by latenitefilms** [reply]
Maybe try trashing your FCP preferences? Maybe you ticked this?

![image](https://github.com/user-attachments/assets/5eb72618-c170-4a57-95f0-e4d3b071661e)

**Reply by JeanSemeur** [reply]
Yes it is what i expect to have ! But no, trashing pref doesn't resolve the problem. Brawtoolbox is still creating a new event. So no need for FCP to ask me if want to Keep both or replace ;)
(as i wrotedifferent app working with FCPXML, i'm a bit aware of FCPXML ;) )

**Comment by JeanSemeur** [comment]
Just make a test.
I export an XML of the synchronised clips in the event "2-11-2024" generated by Braw toolbox then open it with text edit to change the name of the event (replaced by the original name "RUSHES"), then imported the modified XML in FCP and now i've the option to replace or duplicates clips. i choose replace and the clip are replaced and link to the good file on the new disk. So it think there is a problem with braw toolbox by changing the name of the event and not keeping the original one.

**Comment by JeanSemeur** [comment]
Hi Chris, 
Hear a small video to explain my problem with relinking synchronised clip to new disque. Braw Toolbox creates a new event, so it's not a relink but the creation of new clips. New event means you can't REPLACE incoming clips by the new one...
[https://vimeo.com/1026821868/3a4b16bc70]
I'm at the end of a 45 pages of a French documentation about Braw toolbox and i would like to know if a do something wrong. and if not, if you think you are going to address this problem. 
thank for your help Chris
Martin
